.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This file is used by my editor (emacs) and many others to make sure you don't have any indent errors etc...